lib/vagrant-openshift/action/push_openshift_images.rb

```diff
@@ -121,20 +161,20 @@ def call(env)
           # FIXME: We always need to make sure we have the latest base image
           # FIXME: This is because the internal registry is pruned once per month
           if !@options[:build_images].include?("openshift/base")
-            @options[:build_images] = "openshift/base:master,#{@options[:build_images]}"
+            @options[:build_images] = "openshift/base:1:master,#{@options[:build_images]}"
```

Comment:
we now require the version to build as part of the field, and for sti-base we just default to "1"